compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToLLVM.cpp

```diff
@@ -96,7 +96,8 @@ void ConvertToDynamicSharedMemory(ModuleOp moduleOp) {
         loc, globalPtr.getType(), globalPtr, ValueRange({zero, offsetValue}));
     Value castPtr = builder.create<LLVM::BitcastOp>(
```

Comment:
Not related, but I think such bitcasts are not needed when everything is an opaque pointer....

Reply:
Removed BitcastOp